test/integration/patients/worklist/schedule.js

```diff
@@ -579,6 +579,43 @@ context('schedule page', function() {
       .should('be.empty');
   });
 
+  specify('reduced patient schedule employee', function() {
+    cy
+      .server()
+      .routeCurrentClinician(fx => {
+        fx.data.id = '123456';
+        fx.data.attributes.access = 'employee';
+        fx.data.attributes.enabled = true;
+        return fx;
+      })
+      .routeSettings(fx => {
+        const reducedPatientSchedule = _.find(fx.data, setting => setting.id === 'reduced_patient_schedule');
+        reducedPatientSchedule.attributes.value = true;
+
+        return fx;
+      })
+      .routeActions()
+      .visit('/schedule')
```

Comment:
This is correct, but just a note that we should be able to change this to `visit()` when the epic is complete.

Reply:
Okay, good call. Would have probably missed that. Thanks!